src/components/PrintForm/index.tsx

```diff
@@ -120,6 +132,46 @@ export const PrintForm: React.FC<PrintFormProps> = ({
     return false;
   }, [map, layerBlackList]);
 
+  const getPrintableLayers = useCallback((printLayer: LayerType) => {
+    if (!map) {
+      return [];
+    }
+
+    return MapUtil.getAllLayers(map).filter((layer) => {
+      const layerName = layer.get('name');
+      return layerName &&
+        !(layerName.includes('react-geo')) &&
+        layer.getVisible() &&
+        !(layer instanceof OlLayerGroup) &&
+        layer !== printLayer;
+    }) as LayerType[];
+  }, [map]);
+
+  const getAttributions = useCallback((manager: MapFishPrintV3Manager) => {
+    if (!manager) {
```

Comment:
According to the typing this check shouldn't be needed, if this is actually optional, we should mark it accordingly.

Reply:
The unnecessary check got removed.